app/assets/javascripts/controllers/ops/pglogical_replication_form_controller.js

```diff
@@ -228,9 +228,10 @@ ManageIQ.angular.app.controller('pglogicalReplicationFormController', ['$http',
       data["user"] = subscription.user;
       data["password"] = subscription.password;
       data["port"]     = subscription.port;
+      data["id"] = subscription.id
     }
     miqService.sparkleOn();
-    var url = '/ops/pglogical_validate_subscription/' + pglogicalReplicationFormId;
+    var url = '/ops/pglogical_validate_subscription'
```

Comment:
@carbonin @h-kataria right, controller is not expecting ```new``` to be added to url and iIwas testing end-to-end